layouts/tv-episode/single.html

```diff
@@ -41,7 +41,15 @@ <h1 class="h1 mt-4 pt-2"> {{ .Parent.Title -}}</h1>
 <!-- IF UPCOMING -->
 {{ $.Scratch.Set "streamName" "" }}
 {{ $.Scratch.Set "streamUrl" "#" }}
-{{ if isset .Params "twitch" }} 
+{{ if (and (isset .Params "twitch") (isset .Params "youtube")) }}
+  {{ $.Scratch.Set "dualStream" true }}
+  {{ $.Scratch.Set "twitchUrl" (print "https://www.twitch.tv/" .Params.twitch) }}
+  {{ $.Scratch.Set "youtubeUrl" (print "https://www.youtube.com/watch?v=" .Params.youtube) }}
+  {{ $.Scratch.Set "embedSite" .Params.embedSite }}
+  {{ if and (ne ($.Scratch.Get "embedSite") "youtube") (ne ($.Scratch.Get "embedSite") "twitch") }}
```

Comment:
I realize this logic is a bit awkward, but `isset` is showing some odd behavior that this logic helps get around

Reply:
Makes sense if twitch is being used as a fallback in the event that the youtube front matter is empty. If the issue is isset behavior, you could first test the issets on their own conditional before checking for null/empty youtube ids. Theoretically though, line 44 "should" be already doing that if it isn't.